src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,82 @@
+package frc.robot.subsystems;
+
+import com.chopshop166.chopshoplib.logging.LoggedSubsystem;
+import com.chopshop166.chopshoplib.pneumatics.IDSolenoid;
+import com.chopshop166.chopshoplib.pneumatics.MockDSolenoid;
+import com.chopshop166.chopshoplib.sensors.IColorSensor;
+
+import frc.robot.maps.subsystems.IntakeData;
+import edu.wpi.first.wpilibj.DoubleSolenoid.Value;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+public class Intake extends LoggedSubsystem<IntakeData, IntakeData.Map> {
+
+    private IDSolenoid solenoid = new MockDSolenoid();
```

Comment:
We should be passing a `Value` object in the IntakeData object instead of creating the object here. (Yes I told you to do this, yes you do need to change it).